src/main/kotlin/no/nav/familie/ef/personhendelse/handler/ForelderBarnHandler.kt

```diff
@@ -16,13 +26,20 @@ class ForelderBarnHandler(val sakClient: SakClient) : PersonhendelseHandler {
     override val type = PersonhendelseType.FORELDERBARNRELASJON
 
     override fun lagOppgaveBeskrivelse(personhendelse: Personhendelse): String {
-        return "Bruker har fått et nytt barn"
+        var beskrivelse = "Bruker har fått et nytt barn. "
+        val personIdent = personhendelse.identerUtenAktørId().first()
+        val nyeBarnForBruker = sakClient.finnNyeBarnForBruker(PersonIdent(personIdent))
```

Comment:
Hvordan bli dette tro ? Hente to ganger ? Eller er det noen bedre måte mtp disse overridede metodene ?

Reply:
Det kommer sannsynligvis ikke skje mange ganger, sånn sett er det greit. 
Men føles egentlige litt unødvendig? 

Kanskje `skalOppretteOppgave` kan returnere noe annet enn `Boolean` og att det sendes med til `lagOppgaveBeskrivelse` ? 

Eller att "skalOppretteOppgave" er det som returnerer oppgavebeskrivelsen hvis den skal opprette oppgave? 
Sånn att vi kanskje ikke trenger både skalOppretteOppgave og lagOppgaveBeskrivelse ?

Reply:
Ja, hvis vi skal opprette oppgave, så skal vi alltid lage en oppgavebeskrivelse ? Så da kan vi ta oppgavebeskrivelse i samme slengen evt.